typescript-modules/react-modules-quiz-notes.md

```diff
@@ -0,0 +1,43 @@
+# react-intro-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What are modules?
+
+Modules are collections of code in external files
+
+- What is the difference between named and default imports?
+
+Default imports allow for the content to be named in the import, while named imports are named when exported.
+
+- What is the difference between named and default exports?
+
```

Comment:
Technically you are missing this quiz question, but I'll approve it, because I feel like one question kinda answers both of these